.github/workflows/test_suite.yml

```diff
@@ -5,7 +5,40 @@ on:
   pull_request:
 name: SQLAlchemy Spanner dialect
 jobs:
-  tests:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install nox
+        run: python -m pip install nox
+      - name: Run Lint
+        run: nox -s lint_setup_py lint blacken
+
+  unit:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install nox
+        run: python -m pip install nox
+      - name: Run Unit Tests
+        run: nox -s unit
+        env:
+          SPANNER_EMULATOR_HOST: localhost:9010
+          GOOGLE_CLOUD_PROJECT: appdev-soda-spanner-staging
```

Comment:
@larkee let me know if you have a different solution to this. The problem is that the unit tests are not really unit tests. It tries to create an actual Python client underneath, which requires `GOOGLE_APPLICATION_CREDENTIALS` to be set. We have nothing meaningful to set this env var to so the next best thing I could think of was to set the emulator host so that the client defaults to setting anonymous creds.